lib/crypto.js

```diff
@@ -284,28 +285,34 @@ export default class SodiumCrypto extends Crypto {
       {
         name: 'PBKDF2',
         salt: enc.encode(salt),
-        iterations: 1000,
+        iterations,
         hash: 'SHA-512'
       },
       keyMaterial,
       { name: 'HMAC', hash: { name: 'SHA-1' }, length: length * 8 },
       true,
       ['sign', 'verify']
     )
-    window.crypto.subtle.exportKey('raw', key)
+    return window.crypto.subtle.exportKey('raw', key)
   }
 
   /**
    * Derive an Ed25519 keypair from a password and a random salt
    *
-   * @param {string} password User-specified password
-   * @param {string} salt     User-specified salt (should be random)
+   * @param {string} password     User-specified password
+   * @param {string} salt         User-specified salt (should be random)
+   * @param {number} [iterations] Option number of hash iterations to create the seed.
    *
    * @returns {KeyPair} Object containing publicKey and privateKey fields
    */
-  async deriveSigningKey(password, salt) {
+  async deriveSigningKey(password, salt, iterations = 1000) {
```

Comment:
can we pull this 1000 from a default named global?

Reply:
Sure can -- and should!